### PR TITLE
Use @localheinz's composer normalize

### PR DESCRIPTION
**Pull request by WyriHaximus**
To improve package quality

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,9 +14,17 @@
     "friendsofphp/php-cs-fixer": "^2.5",
     "jakub-onderka/php-console-highlighter": "^0.3.2",
     "jakub-onderka/php-parallel-lint": "^0.9.2",
+    "localheinz/composer-normalize": "^0.1.0",
     "phake/phake": "^3.0.1",
     "phpunit/phpunit": "^6.2"
   },
+  "require-dev": {},
```
Comment (localheinz) [inline composer.json]: Was thinking, maybe I should add a normalizer that removes empty hashes, at least on a root level!

Comment (WyriHaximus) [reply composer.json]: That would be very welcome 👍

File: composer.json [hunk 2]
```diff
@@ -14,9 +14,17 @@
     "friendsofphp/php-cs-fixer": "^2.5",
     "jakub-onderka/php-console-highlighter": "^0.3.2",
     "jakub-onderka/php-parallel-lint": "^0.9.2",
+    "localheinz/composer-normalize": "^0.1.0",
```
Comment (harikt) [inline composer.json]: this should go in require-dev right ?

Comment (WyriHaximus) [reply composer.json]: Normally yes, but this package is a meta package to be added to `require-dev` it self.

**Pull request merged**